packages/yew/src/scheduler.rs

```diff
@@ -46,15 +46,6 @@ impl TopologicalQueue {
     }
 
     /// Take a single entry, preferring parents over children
-    #[rustversion::before(1.66)]
-    fn pop_topmost(&mut self) -> Option<QueueEntry> {
-        // BTreeMap::pop_first is available after 1.66.
-        let key = *self.inner.keys().next()?;
-        self.inner.remove(&key)
-    }
-
-    /// Take a single entry, preferring parents over children
-    #[rustversion::since(1.66)]
     #[allow(clippy::incompatible_msrv)]
```

Comment:
I think you can remove this suppression of the false positive of `clippy::incompatible_msrv` now.

Reply:
```suggestion
```